trade-finance-manager-ui/templates/case/tasks/_macros/tasks-table.njk

```diff
@@ -31,7 +31,11 @@
 
                 <td class="govuk-table__cell govuk-!-padding-top-4 govuk-!-padding-right-4 govuk-!-padding-bottom-4 govuk-!-padding-left-0 ukef-tasks-table__cell-width-28">
                   {# checks if can edit and if user is in tasks team id group #}
-                  {% if task.canEdit and task.team.id in user.teams %}
+                  {% set isUserInTaskTeam = false %}
+                  {% for team in user.teams %}
+                    {% set isUserInTaskTeam = isUserInTaskTeam or team.id === task.team.id %}
+                  {% endfor %}
+                  {% if task.canEdit and isUserInTaskTeam %}
```

Comment:
This feels like something we're going to want to do often. This can't be resolved with the middleware method (as you've done for the PDC users) so maybe we need a custom function we can call in these cases. e.g. so this could instead be:
```
{% set isUserInTaskTeam = user.teams | hasTeamId(task.team.id) %}
```
What do you think?

Reply:
We would probably need to add the `userIsInTeam` function to nunjucks via this kind of approach https://stackoverflow.com/questions/51788055/javascript-function-in-nunjucks